GlucoseOnFhir/GlucoseMeterViewController.swift

```diff
@@ -6,77 +6,85 @@
 //  Copyright © 2016 CocoaPods. All rights reserved.
 //
 
-//TO-DO: only add observations to the bundle that do not exist on the FHIR server
-
 import Foundation
 import UIKit
 import CoreBluetooth
 import CCGlucose
 import CCToolbox
-import SwiftSpinner
 import SMART
 
-class GlucoseMeterViewController: UITableViewController, GlucoseProtocol, FHIRProtocol {
+class GlucoseMeterViewController: UITableViewController, GlucoseProtocol {
     private var glucose : Glucose!
     let cellIdentifier = "GlucoseMeterCellIdentifier"
+    var glucoseMeterConnected: Bool! = false
+    var glucoseMeter: CBPeripheral!
     var glucoseFeatures: GlucoseFeatures!
     var glucoseMeasurementCount: UInt16 = 0
     var glucoseMeasurements: Array<GlucoseMeasurement> = Array<GlucoseMeasurement>()
     var glucoseMeasurementContexts: Array<GlucoseMeasurementContext> = Array<GlucoseMeasurementContext>()
     var selectedGlucoseMeasurement: GlucoseMeasurement!
     var selectedGlucoseMeasurementContext: GlucoseMeasurementContext!
-    var selectedMeter: CBPeripheral!
-    var meterConnected: Bool!
-    private var fhir : FHIR!
+    var log: [[String]] = []
```

Comment:
Instead of a mysterious array of arrays, it would be much clearer to use a simple struct. e.g.

```
struct LogMessage {
    let date: Date
    let text: String
}

var log:[LogMessage] = []
```